ViewModel/BrowserViewModel.swift

```diff
@@ -179,7 +180,7 @@ class BrowserViewModel: NSObject, ObservableObject,
         } else if url.isKiwixURL {
             decisionHandler(.allow)
         } else if url.scheme == "http" || url.scheme == "https" {
```

Comment:
If you are introducing `isExternalLink` on `URL`, could you change this line to use that variable instead?